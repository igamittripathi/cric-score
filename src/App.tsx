import React from 'react';
import './App.css';
import { TeamCreater } from './containers'

function App() {
  return (
    <div className="App">
     <TeamCreater/>
    </div>
  );
}

export default App;
