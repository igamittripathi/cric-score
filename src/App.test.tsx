import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe('App', () => {
  let wrapped;
  beforeEach(()=>{
    wrapped = shallow(<App/>);
  })
  
  test('should render Component correctly', () => {   
    expect(wrapped).toMatchSnapshot();
  });

  test('renders the div',()=>{
    expect(wrapped.find('div').hasClass('App')).toEqual(true);
  });
});