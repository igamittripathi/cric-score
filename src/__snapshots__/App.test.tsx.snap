// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render Component correctly 1`] = `
<div
  className="App"
>
  <AppRoute />
</div>
`;

exports[`App should render the Title Component correctly 1`] = `
<div
  className="App"
>
  <AppRoute />
</div>
`;
